caproto/server/common.py

```diff
@@ -566,14 +580,19 @@ def get_db_entry():
             self.events_on.clear()
             self.circuit.log.info("Client at %s:%d has turned events off.",
                                   *self.circuit.address)
+            to_send = []
         elif isinstance(command, ca.ClearChannelRequest):
             chan, db_entry = get_db_entry()
             await self._cull_subscriptions(
                 db_entry,
                 lambda sub: sub.channel == command.sid)
-            return [chan.clear()]
+            to_send = [chan.clear()]
         elif isinstance(command, ca.EchoRequest):
-            return [ca.EchoResponse()]
+            to_send = [ca.EchoResponse()]
+        if isinstance(command, ca.Message):
+            tags['bytesize'] = len(command)
+            self.log.debug("%r", command, extra=tags)
+            return to_send
```

Comment:
Was this meant to be one level un-indented?  It looks like this went from returning a list always to sometimes returning `None` and sometimes a list.

Reply:
Good catch! Fix pushed.